metaseq/__init__.py

```diff
@@ -18,7 +18,8 @@
 
 # backwards compatibility to support `from metaseq.X import Y`
 from metaseq.distributed import utils as distributed_utils
-from metaseq.logging import meters, metrics, progress_bar  # noqa
+from metaseq.logging import meters, metrics  # noqa
+from .logging.progress_bar import base_progress_bar
```

Comment:
Can we avoid relative imports?

Reply:
Yeah in the process of fixing. Also getting rid of this module rename for progress bars.